zap/src/main/java/org/zaproxy/zap/spider/filters/ParseFilter.java

```diff
@@ -71,6 +71,9 @@ public FilterResult filtered(HttpMessage responseMessage) {
         /** Indicates that the resource was not filtered. */
         public static final FilterResult NOT_FILTERED = new FilterResult();
 
+        /** Indicates that the resource is wanted by a custom parser. */
```

Comment:
Worth explaining that this skips the default filter, if no other filter filters it out.